packages/agents-copilotstudio-client/package.json

```diff
@@ -19,10 +19,22 @@
   ],
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
+  "browser": {
+    "os": "./src/browser/os.ts",
+    "crypto": "./src/browser/crypto.ts"
+  },
+  "scripts": {
+    "build:browser": "esbuild --platform=browser --target=es2019 --format=esm --bundle --sourcemap --minify --outfile=dist/src/browser.mjs src/index.ts"
+  },
   "dependencies": {
     "@microsoft/agents-activity": "file:../agents-activity",
     "axios": "^1.9.0",
-    "debug": "^4.3.7"
+    "rxjs": "5.5.12",
+    "uuid": "^11.1.0"
+  },
+  "devDependencies": {
```

Comment:
we are keeping all devDependencies in the /package.json at the workspace root.

Curious to understand why `debug` is declared as devDependency